src/library-authoring/containers/ContainerOrganize.tsx

```diff
@@ -0,0 +1,84 @@
+import { useMemo, useState } from 'react';
+import { useIntl } from '@edx/frontend-platform/i18n';
+import { getConfig } from '@edx/frontend-platform';
+import { Collapsible, Icon, Stack } from '@openedx/paragon';
+import {
+  ExpandLess, ExpandMore, Tag,
+} from '@openedx/paragon/icons';
+
+import { ContentTagsDrawer, useContentTaxonomyTagsData } from '../../content-tags-drawer';
+import { useLibraryContext } from '../common/context/LibraryContext';
+import { useSidebarContext } from '../common/context/SidebarContext';
+import messages from './messages';
+
+const ContainerOrganize = () => {
+  const intl = useIntl();
+  const [tagsCollapseIsOpen, setTagsCollapseOpen] = useState(true);
+
+  const { readOnly } = useLibraryContext();
+  const { sidebarComponentInfo } = useSidebarContext();
+
+  const containerId = sidebarComponentInfo?.id;
+  // istanbul ignore if: this should never happen
+  if (!containerId) {
+    throw new Error('containerId is required');
+  }
+
+  const { data: componentTags } = useContentTaxonomyTagsData(containerId);
+
+  const tagsCount = useMemo(() => {
+    if (!componentTags) {
+      return 0;
+    }
+    let result = 0;
+    componentTags.taxonomies.forEach((taxonomy) => {
+      const countedTags : string[] = [];
+      taxonomy.tags.forEach((tagData) => {
+        tagData.lineage.forEach((tag) => {
+          if (!countedTags.includes(tag)) {
+            result += 1;
+            countedTags.push(tag);
+          }
+        });
+      });
+    });
+    return result;
+  }, [componentTags]);
+
+  return (
+    <Stack gap={3}>
+      {[true, 'true'].includes(getConfig().ENABLE_TAGGING_TAXONOMY_PAGES)
+        && (
+          <Collapsible.Advanced
+            open={tagsCollapseIsOpen}
+            className="collapsible-card border-0"
+          >
+            <Collapsible.Trigger
+              onClick={() => setTagsCollapseOpen((prev) => !prev)}
```

Comment:
Could you add test coverage for this line since you're already almost perfect?

Reply:
Sure! Fixed here: [0b56216](https://github.com/openedx/frontend-app-authoring/pull/1782/commits/0b56216ec3e2da92b6dd3fcaaad577079b27c328)